ext/mbstring/php_mbregex.c

```diff
@@ -1176,6 +1169,7 @@ static void _php_mb_regex_ereg_replace_exec(INTERNAL_FUNCTION_PARAMETERS, OnigOp
 					zval_ptr_dtor(&retval);
 				} else {
 					if (!EG(exception)) {
+						// TODO Is this a return throw?
```

Comment:
Throwing here should be fine. We'll probably want to change zend_call_function() to always throw, but the last time I looked into this it was not entirely simple ... because reasons.